lib/themes/dosomething/paraneue_dosomething/scss/patterns/_container.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+  &.-hidden {
```

Comment:
Can we consider renaming this `.optimizely-hidden` or something similar to denote that it's for a temporary A/B test?